src/components/ui/Tag.astro

```diff
@@ -15,11 +16,12 @@ const colorMap = {
 const {
   color = "semidarkblue",
   classList = [],
+  href = null,
 } = Astro.props;
 
-const localClassList = `${colorMap[color]} rounded-lg font-bold text-white text-sm px-[0.5rem] py-[0.25rem]`;
+const localClassList = `${colorMap[color]} w-fit rounded-lg font-bold !text-white !no-underline text-sm px-[0.5rem] py-[0.25rem]`;
 ---
 
-<span class:list={[localClassList, classList]}>
+<a href={href} class:list={[localClassList, classList]}>
   <slot/>
-</span>
+</a>
```

Comment:
I guess we should render either span or a if href is present or not since especially accesibility tools such as screenreaders will always communicate there is an anchor tag present even if it is not one.

```
{href ? <a ..../> : <span ...../>}

Reply:
Good idea